### ISSUE TITLE
External executables enable all esp files.

### ISSUE THREAD

**Issue by Piperman132** [TanninOne/modorganizer#1012]
When ever I run an external executable file such as "LOOT.exe" or "Wrye Bash.exe" all esps get enabled in Mod Organizer. 

Using 2.0.7 Beta 3, Loot 0.10 and WryeBash 307.36. Running Loot or WryeBash and then closing them enables all esp files for Skyrim Special Edition. I'm using the plugin from here "http://www.nexusmods.com/skyrimspecialedition/mods/1725/?tab=1&navtag=http%3A%2F%2Fwww.nexusmods.com%2Fskyrimspecialedition%2Fajax%2Fmoddescription%2F%3Fid%3D1725%26preview%3D&pUp=1".

Sorry if I've done something incorrect on here, I'm new to github.

**Comment by GSDFan** [comment]
Yes, this is a bug in MO 2. It happens with Fallout 4 also.

**Comment by Piperman132** [comment]
Ah, ok, thanks for the heads up. I take it there is no work around either then?

**Comment by Akzyra** [comment]
Yes, wait for the plugin/new MO2 by [this guy](http://www.nexusmods.com/skyrimspecialedition/mods/1725).

**Comment by GSDFan** [comment]
The plugin helps, but the program itself needs to be updated to fully support the game.
